src/Select/SelectInput.js

```diff
@@ -0,0 +1,72 @@
+import React from 'react'
+import propTypes from 'prop-types'
+import cx from 'classnames'
+import { ArrowDown, ArrowUp } from '../icons/Arrow.js'
+import { colors } from '../theme.js'
+
+const SelectInput = ({ placeholder, selected, open, clearable, onClear }) => {
+    const Arrow = open ? ArrowUp : ArrowDown
+    const hasSelection = 'label' in selected
+
+    return (
+        <React.Fragment>
+            <div className="input">
+                <span className={cx({ placeholder: !hasSelection })}>
+                    {hasSelection ? selected.label : placeholder}
+                </span>
+                <span className="right">
+                    {clearable && hasSelection && (
+                        <a className="clear" onClick={onClear}>
```

Comment:
This should be a button, not a link

Reply:
Yes, I agree. I encountered this before with the scheduler-app as well, where the design asked for an action trigger that looks like a link. But since it doesn't trigger navigation I think it should be a button.

I would like to introduce a component for this since I've had this problem in more areas than just here. The button looks like a link, but is a button element. So that we can be both correct semantically and for the design. What do you think?

I can imagine that the counter argument would be, why would you want buttons to look like links. This follows from the design. I think that if in our design we want actions to be triggered  by components that look like links, we should have buttons that look like links. If we don't want to build those, we could also modify the design to make them look like buttons, that could also be a solution. Or we could just not care about the semantics, but I would not prefer that.

Reply:
Agree.
@cooper-joe Can we have buttons that look like links?

Reply:
Note to @cooper-joe, we're asking for this mostly because of semantics. In the designs we have components that looks like traditional links, but perform an action without navigating. Semantically it'd be more correct to have that be a button underneath, which is styled to look like a link. The semantics ultimately matter for accessibility, where browsers will handle elements differently based on their types (i.e., link or button).

Reply:
Yes, 'text' is a valid button type, but I need to clarify where it should be used. It's a murky issue. Upon reading this I'm thinking let's change the clear function to a button. Specifically a secondary, small button from ui-core.

It will look like this:
![image](https://user-images.githubusercontent.com/33054985/66298955-e26c7880-e8f2-11e9-846e-c69df506b93b.png)

Reply:
That looks good 👍

Maybe we should decide this in general as well. As in, all action triggers that don't navigate should be buttons? That would completely eliminate this problem as well, and seems like a very straightforward solution (also for the user, since things are then what they look like).